### PR TITLE
Add getting-to-know-us interview with Paulo F. Oliveira

### PR DESCRIPTION
**Pull request by starbelly**
- [x] I acknowledge my contribution to the website does not assert comparative or superlative differences of one product, project, company or individual over another.

# Screenshots

Screen Shot 2021-07-14 at 7 01 38 PM

### CODE REVIEW

File: priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md [hunk 1]
```diff
@@ -0,0 +1,171 @@
+{
+  "title": "Getting to know - Paulo F. Oliveira",
+  "authors": ["The Marketing Working Group"],
+  "slug": "getting-to-know-paulo-oliveira",
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md:4]:
```suggestion
  "slug": "getting-to-know-paulo-f-oliveira",
```

File: priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+I left Santo André to go to the University of Algarve, where I studied
+Computer Systems Engineering and where I got really interested in
+software development. It is also where I realized that it was not as
+difficult as I thought it was. My final thesis was a parser/lexer for
+Fortran targeting parallel image processing on Beowulf Clusters
+(SPMDdir), for which I wrote a paper on.
+I then went to work for SAP. About one and a half years later, I quit: I
+became more interested in software development there but not equally
+interested in the product.
+I started learning programming by myself (it was not the main focus of
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md:47]:
```suggestion
I continued learning programming by myself (it was not the main focus of
```

File: priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md [hunk 3]
```diff
@@ -28,7 +28,7 @@ Paulo F. Oliveira is a Portuguese software developer and a member of the
 Erlang Ecosystem Foundation. With a career spanning over 17 years, he
 has worked on the most diverse projects: from mobile games backends
 and internet portals, to NATO port surveillance systems and helicopter
-engine software, to space projects. And today, we get to know him.
+engine software, to space projects. And today, we got know him.
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md:31]: This fix doesn't seem right. You mean "got to"?

File: priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md [hunk 4]
```diff
@@ -77,24 +77,24 @@ I got my first experience with high scalability systems and millions of
 concurrent connections: they were building a system that was similar to
 Erlang+WebSockets, and this was the first time I heard about this
 language. I was even told that if you spend enough time working around a
-system that needs full tolerance, scalability, and high concurrence, you
+system that needs fault tolerance, scalability, and high concurrency, you
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md:80]: This flew by me 😄

File: priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md [hunk 5]
```diff
@@ -128,7 +128,7 @@ absolutely use it always.
 **Are you learning something new at the moment?**
 
 I started learning Elixir a few months ago for two main reasons: the
-first one is that is not very easy to recruit Erlang-first developers,
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md]: I don't think this fix is correct.

File: priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md (hunk 3 above)
Comment (starbelly) [reply priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md:31]: Fine catch 👍

File: priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md [hunk 6]
```diff
@@ -0,0 +1,171 @@
...
+and it was super slow: concurrency is a big part of what I love, but I
+also love performance, and Erlang gives me that.
+As a CI and static analysis aficionado, one other thing I have to
+mention that was really nice to find out-of-the-box was dialyzer. I
+absolutely use it always.
+
+**Are you learning something new at the moment?**
+
+I started learning Elixir a few months ago for two main reasons: the
+first one that is not very easy to recruit Erlang-first developers,
```
Comment (paulo-ferraz-oliveira) [inline priv/posts/marketing/20210714233950_getting-to-know-paulo-oliveira.md:131]:
```suggestion
first one is that it is not very easy to recruit Erlang-first developers,
```

**Review by paulo-ferraz-oliveira — APPROVED** [review]

**Pull request merged**